Tasks/AzureRmWebAppDeploymentV4/Tests/WindowsWebAppRunFromZipProviderL0Tests.ts

```diff
@@ -0,0 +1,76 @@
+import tl = require('vsts-task-lib');
+import { DeploymentFactory } from '../deploymentProvider/DeploymentFactory';
+import { WindowsWebAppRunFromZipProvider } from '../deploymentProvider/WindowsWebAppRunFromZipProvider'
+import { IWebAppDeploymentProvider } from '../deploymentProvider/IWebAppDeploymentProvider';
+import { TaskParametersUtility, TaskParameters, DeploymentType } from '../operations/TaskParameters';
+import { stringify } from 'querystring';
+import { PackageType } from 'webdeployment-common/packageUtility';
+import { getMockEndpoint } from '../node_modules/azure-arm-rest/Tests/mock_utils';
+import { mockAzureARMPreDeploymentSteps, mockRunFromZipSettings }  from "./mock_utils";
+
+getMockEndpoint();
+mockAzureARMPreDeploymentSteps();
+mockRunFromZipSettings();
+
+export class WindowsWebAppRunFromZipProviderL0Tests  {
+
+    public static async startWindowsWebAppRunFromZipProviderL0Tests() {
+        await WindowsWebAppRunFromZipProviderL0Tests.testForPreDeploymentSteps_RunFromZipProvider();
+        await WindowsWebAppRunFromZipProviderL0Tests.testForPreDeploymentStepsWithSlotEnabled_RunFromZipProvider();
```

Comment:
with folder package type

Reply:
Done. Added in this commit fdd57b62de16f1d6f2671cd79cf1988304381151